braph2genesis/graphs/_OrdMlWU.gen.m

```diff
@@ -0,0 +1,259 @@
+%% ¡header!
+OrdMlWU < Graph (g, ordinal multilayer weighted undirected graph) is an ordinal multilayer weighted undirected graph.
+
+%%% ¡description!
+In an ordinal multilayer weighted undirected (WU) graph, layers have different number 
+ of nodes with within-layer weighted undirected edges, associated with a real 
+ number between 0 and 1 and indicating the strength of the connection.
+The connectivity matrices are symmetric.
+There are connections between layers connecting the corresponding nodes.
+The layers are connected in an ordinal fashion, i.e., only consecutive layers are connected.
```

Comment:
this description is wrong, please correct based on the updated in this PR: https://github.com/giovannivolpe/BRAPH-2-Matlab-beta/pull/9